packages/framework/src/Form/Admin/LegalConditions/TermsAndConditionsSettingFormType.php

```diff
@@ -46,8 +50,40 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'help' => t('Choose article, which will serve as terms and conditions with which the customer has to agree when creating order.'),
             ]);
 
+        $builderWithdrawalGroup = $builder->create('withdrawal', GroupType::class, [
+            'label' => t('Withdrawal from a contract'),
+        ]);
+
+        $builderWithdrawalGroup
+            ->add('withdrawalDeadlineDays', IntegerType::class, [
+                'required' => true,
+                'label' => 'Withdrawal deadline (days)',
+                'constraints' => [
+                    new Constraints\NotBlank([
+                        'message' => 'Please enter withdrawal deadline.',
+                    ]),
+                    new Constraints\Positive([
+                        'message' => 'Withdrawal deadline must be a positive number.',
+                    ]),
+                ],
+            ])
+            ->add('withdrawalInstructions', CKEditorType::class, [
+                'required' => true,
```

Comment:
can we please add help text? I have absolutely no idea why I should set it and where is it then displayed.

Reply:
:+1: a4d494e098